packages/frontend/src/creation-form/components/approve-rewards.tsx

```diff
@@ -123,7 +123,9 @@ export const ApproveRewards = ({
         !currentlyApprovingReward ||
         loading ||
         !spender ? (
-        <Button loading>Loading</Button>
+        <Button size="small" loading className={{ root: "w-full" }}>
```

Comment:
FYI, no issue here but this translation shouldn't really be required as no text is shown when the button is in the loading state, and here it's always always loading. Initially the button had no text but I had to add some because the height of the button changed and it became shorter, which is actually an error in the button component itself (if no text is specified but the loading state is, the button shouldn't shrink).